scripts/fixtures/eform/test-pattern.html

```
@@ -0,0 +1,605 @@
+<!DOCTYPE html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)